gm4_combat_expanded/data/gm4_combat_expanded/loot_tables/mob/equip_arrow/stray_toxic.json

```diff
@@ -15,110 +15,75 @@
-                  "italic": false,
-                  "extra": [
+                  "translate": "potion.withDuration",
+                  "with": [
                     {
-                      "translate": "%1$s%3427655$s",
+                      "translate": "potion.withAmplifier",
                       "with": [
                         {
-                          "text": " (0:05)"
+                          "translate": "effect.minecraft.poison"
                         },
                         {
-                          "translate": "item.gm4.combat_expanded.lore.duration_5"
+                          "translate": "potion.potency.1"
                         }
-                      ],
-                      "color": "red",
-                      "italic": false
-                    }
-                  ]
-                },
-                {
-                  "text": ""
+                      ]
+                    },
+                    "0:05"
+                  ],
+                  "color": "red",
+                  "italic": false
                 },
                 {
-                  "translate": "%1$s%3427655$s",
+                  "translate": "potion.withDuration",
                   "with": [
                     {
-                      "text": "When Applied:"
+                      "translate": "effect.minecraft.weakness"
                     },
-                    {
-                      "translate": "item.gm4.combat_expanded.lore.when_applied"
-                    }
+                    "0:05"
                   ],
+                  "color": "red",
+                  "italic": false
+                },
+                {
+                  "text": ""
+                },
+                {
+                  "translate": "potion.whenDrank",
                   "color": "purple",
                   "italic": false
                 },
                 {
-                  "translate": "%1$s%3427655$s",
+                  "translate": "attribute.modifier.take.1",
                   "with": [
                     {
-                      "text": "-4 Attack Damage"
+                      "text": "15"
                     },
                     {
-                      "translate": "item.gm4.combat_expanded.lore.weakness_applied"
+                      "translate": "attribute.name.generic.movement_speed"
                     }
                   ],
                   "color": "red",
                   "italic": false
                 },
                 {
-                  "translate": "%1$s%3427655$s",
+                  "translate": "attribute.modifier.take.0",
                   "with": [
                     {
-                      "text": "-15% Speed"
+                      "text": "4"
                     },
                     {
-                      "translate": "item.gm4.combat_expanded.lore.slowness_applied"
+                      "translate": "attribute.name.generic.attack_damage"
```

Comment:
Did these modifiers get swapped? I don't think it matters tho?

Reply:
yes, I swapped them to match the order of the effects above